packages/components/src/VirtualizedList/VirtualizedList.scss

```
@@ -25,6 +25,15 @@
 	}
 }
 
+:global(div[role='columnheader']) {
```

```suggestion
:global(div[role='columnheader']) {
```
Why do we need :global for `div` and a role? Are they not global by nature?